Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -773,7 +773,7 @@ def test_pull
     end
 
     assert_match "Testing URLs require `--bottle`!",
-      cmd_fail("pull", "http://bot.brew.sh/job/Homebrew\%20Testing/1028/")
+      cmd_fail("pull", "https://bot.brew.sh/job/Homebrew\%20Testing/1028/")
```

Comment:
this build isn't in the jenkins cache anymore

* https://bot.brew.sh/job/Homebrew%20Testing/